docs/tutorial.rst

```diff
@@ -114,6 +114,107 @@ details, but this *is* pretty cool:
 For easier indentation, Shift+Tab will enter 4 spaces.
 And that about wraps it up for the basics section.  It is just like Python.
 
+Customizing the Prompt
+======================
+Customizing the prompt is probably the most common reason for altering an
+environment variable.  The ``PROMPT`` variable can be a string, or it can be a
+function (of no arguments) that returns a string.  The result can contain
+keyword arguments, which will be replaced automatically:
```

Comment:
> The result can contain keyword arguments, which will be replaced automatically

This sentence make little sens – I see what you mean, the string will be formatted using `.format` and keyword arguments, – but that sounds off. 

I though have no better phrasing.